### PR TITLE
webpack: load modules from NODE_PATH too

### PR DESCRIPTION
**Pull request by nkzawa**
support:

```sh
$ NODE_PATH=/my_node_modules next

# need to set NODE_PATH on both `build` and `start`
$ NODE_PATH=/my_node_modules next build
$ NODE_PATH=/my_node_modules next start
```

### CODE REVIEW

**Comment by impronunciable** [comment]
Looks good to me @nkzawa

File: server/build/webpack.js [hunk 1]
```diff
@@ -152,7 +152,7 @@ export default async function createCompiler (dir, { hotReload = false } = {}) {
       root: [
         nodeModulesDir,
         join(dir, 'node_modules')
-      ]
+      ].concat((process.env.NODE_PATH || '').split(';'))
```
Comment (arunoda) [inline server/build/webpack.js]: May be we need to use a split separator like this: 

```
process.platform === 'win32' ? ';' : ':'
```

This is to support windows. CRA and Storybook also follows this pattern.

Comment (nkzawa) [inline server/build/webpack.js]: 👍

**Comment by nkzawa** [comment]
fixed!

**Pull request merged**